### DISCUSSION TITLE
Simulating a nucleoprotein using MARTINI_v2

### DISCUSSION THREAD

**Discussion by satyajitkhatua09** [Martini-Force-Field-Initiative/MartiniDiscussions#2] [Nucleic acids]
I have a nucleoprotein that was parametrized with the Martinize Python scripts for protein and DNA (as in Advanced tutorial: protein-DNA complex](https://cgmartini.nl/docs/tutorials/Legacy/martini2/dna.html). But while running the simulations, It stuck in the equilibration step with the error log attached. The system with protein only is running okay. 

As this is the first time I am using CG simulation and also using GROMACS MD engine, I find it very difficult to solve the errors. The following is the equilibration input:

integrator               = md 
dt                       = 0.010
nsteps                   = 200000

nstxout			 = 0
nstvout                  = 0
nstfout                  = 0
nstlog                   = 1000
nstenergy                = 100
nstxout-compressed	 = 1000
compressed-x-precision   = 100

cutoff-scheme            = Verlet
nstlist                  = 10
ns_type                  = grid
pbc                      = xyz
verlet-buffer-tolerance  = 0.005

coulombtype              = reaction-field
rcoulomb                 = 1.1
epsilon_r                = 15
vdw_type                 = cutoff
vdw-modifier             = Potential-shift-verlet
rvdw                     = 1.1

tcoupl                   = v-rescale 
tc-grps                  = System 
tau_t                    = 2.0 ;0.5 
ref_t                    = 300 
Pcoupltype               = isotropic
Pcoupl                   = berendsen 
tau_p                    = 3.0      
compressibility          = 3e-4    
ref_p                    = 1.0                                             
refcoord-scaling         = com

constraints		 = none

[equil.log](https://github.com/user-attachments/files/16775846/equil.log)

**Comment by cgmartini** [comment]
Your initial temperature seems to be very low (close to 0 K). Better to provide initial velocities (gen_vel = yes).
